src/datasources/targeted-messaging/outreach-file-processor.spec.ts

```diff
@@ -255,6 +279,7 @@ describe('OutreachFileProcessor', () => {
         .with('sourceId', outreachFile.campaign_id)
         .with('sourceFile', sourceFile)
         .with('sourceFileChecksum', fileChecksum)
+        .with('targetAll', false)
```

Comment:
As the previous comments: Question: why is it added? Does it make a difference if it's false or true here?

Reply:
similar here, if it is true it will skip processing the targetedSafes in the test wont be created